lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
```

Comment:
It sounds to me like it needs to be a little higher in the DOM on this page then.  It's more important that every page have at least one main element and its in a logical place.